### ISSUE TITLE
[demo][nav][accessibility] 

### ISSUE THREAD

**Issue by yanis-git** [docaxess/webcomponents#47]
Here is few point i have notice:
1. H3 with button is valid but not consider as best practice. 
2. class OVERVIEW in uppercase is not consider as best practice.
3. aria-expended without aria control + aria-expended should be on the UL not on the controller button.
4. no aria-label or any way to explain the behavior.

will be good to get complete feedback from accessibility team

![Capture d’écran 2024-09-04 à 10 09 48](https://github.com/user-attachments/assets/69f053f5-55d1-4c3f-8326-9c98a58f0c19)

---

in a nutshell, why not use ip-accordion here ? :P
